src/lpython/semantics/python_ast_to_asr.cpp

```diff
@@ -5533,7 +5531,11 @@ class BodyVisitor : public CommonVisitor<BodyVisitor> {
                 ASR::asr_t* assign = ASR::make_Assignment_t(al, x.base.base.loc,
                                 ASRUtils::EXPR(ASR::make_Var_t(al, x.base.base.loc, tmp_assign_variable_sym)),
                                 target, nullptr);
-                current_body->push_back(al, ASRUtils::STMT(assign));
+                if (current_body) {
```

Comment:
Let's be explicit here

```suggestion
                if (current_body != nullptr) {
```

This way we get to know that `current_body` is a pointer (and not an integer or any other type).